### PR TITLE
Uvec test

### PR DESCRIPTION
**Pull request by brunozc**

### CODE REVIEW

File: tests/test_newmark.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from uvec_ten_dof_vehicle_2D.newmark_solver import NewmarkExplicit
+
+
+def test_Newmark():
```
Comment (aronnoordam) [inline tests/test_newmark.py]: add description

File: tests/analytical_solutions/moving_vehicle.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
+import numpy as np
+
+
+class TwoDofVehicle:
+    """
+    Computes the displacement of a two DOF vehicle crossing a simple supported beam.
+    Displacement computed at middle spam. Solution assuming only first mode.
+
+    Based on Biggs "Introduction do Structural Dynamics", pp pg 322
```
Comment (aronnoordam) [inline tests/analytical_solutions/moving_vehicle.py:9]: missing attributes in docstring

File: tests/analytical_solutions/moving_vehicle.py
Comment (aronnoordam) [inline tests/analytical_solutions/moving_vehicle.py:1]: all docstrings in this file have a different format compared to the rest of the code:

for example, it shoul should be :
`
Args: 
`
`
    - t_step (float): time step (default 0.001)
`

File: uvec_two_dof_vehicle_2D/uvec.py
Comment (aronnoordam) [inline uvec_two_dof_vehicle_2D/uvec.py:1]: why dont you use the existing newmark solver?

Comment (brunozc) [reply uvec_two_dof_vehicle_2D/uvec.py:1]: Because I was checking why we were getting different results and then i tried this.

**Review by aronnoordam — APPROVED** [review]
For now I approve, but in the future a refactor of the package structure is required.  #4

**Pull request merged**